### PR TITLE
Error component

### PR DESCRIPTION
**Pull request by timkeresey**
### What does this PR do?

- Sets up basic structure of Error component.

### Where should the reviewer start?

- src/Error/Error.tsx

### How should this be manually tested?

- testing file to come

### Any background context you want to provide?

- none

### What are the relevant tickets?

- #6 

### Questions:

- none

### CODE REVIEW

File: src/components/Error/Error.tsx [hunk 1]
```diff
@@ -1,11 +1,18 @@
...
-
 const Error: React.FC = () => {
 
   return (
-
+    <div>
+      <h1>404error</h1>
+      <div>
+        Menu
+      </div>
```
Comment (Atos20) [inline src/components/Error/Error.tsx]: one thing I am having a hard time with is making the CSS variables accessible to any Css file. Right now the variables are helping to make sure we use the same font style and color across the application. Do you know how to 'import ' these variables to other files?

**Review by Atos20 — APPROVED** [review]
Great Start Ti

File: src/components/Error/Error.tsx (hunk 1 above)
Comment (EHale64) [reply src/components/Error/Error.tsx]: Are we importing the same CSS everywhere or just for specific files?

**Review by EHale64 — APPROVED** [review]

File: src/components/Error/Error.tsx (hunk 1 above)
Comment (Atos20) [reply src/components/Error/Error.tsx]: Ideally everywhere.

**Pull request merged**